libs/logic-apps-shared/src/designer-client-services/lib/connector.ts

```diff
@@ -12,6 +12,7 @@ export interface ListDynamicValue {
 export interface TreeDynamicValue {
   value: any;
   displayName: string;
+  fullyQualifiedDisplayName?: string;
```

Comment:
What does this mean? what is displayName and what is fullyQualifiedDisplayName?? only one is displayed in UI so stamp it in displayName before sending back to designer from service

Reply:
See test file for examples.

Reply:
Since the test file is now removed, here's an example:

![image](https://github.com/user-attachments/assets/1b01089f-e2e0-4898-a30f-845c3a763f87)

Reply:
Do we still need the fullyQualified display name?

Reply:
Only for completeness; it is referenced but only as an untyped string:

![image](https://github.com/user-attachments/assets/a1af3602-97da-4dbe-b2f7-f213fb95b8a9)

Reply:
Then is the service reading it correctly and setting it? I dont see it getting used in any UI component

Reply:
Yeah, it's read back here:

![image](https://github.com/user-attachments/assets/3ef9169a-15db-48b9-ae4b-be4a82a57e5e)

`libs\designer\src\lib\ui\panel\nodeDetailsPanel\tabs\parametersTab\index.tsx`